app/views/course/experience_points_records/new.html.slim

```diff
@@ -0,0 +1,5 @@
+= page_header
+
+h2 = @course_user.name
```

Comment:
I think this is more informative if you say New Experience Points Record for %{name}